packages/validator/src/services/attestation.ts

```diff
@@ -135,9 +135,11 @@ export class AttestationService {
     // never beyond the 1/3 cutoff time.
     // https://github.com/status-im/nimbus-eth2/blob/7b64c1dce4392731a4a59ee3a36caef2e0a8357a/beacon_chain/validators/validator_duties.nim#L1123
     const msToOneThirdSlot = this.clock.msToSlot(slot + 1 / 3);
-    // Default = 1/6, which is half of attestation offset
-    const afterBlockDelayMs = 1000 * this.clock.secondsPerSlot * (this.opts?.afterBlockDelaySlotFraction ?? 1 / 6);
-    await sleep(Math.min(msToOneThirdSlot, afterBlockDelayMs));
+    // submitting attestations asap to avoid busy time at around 1/3 of slot
+    const afterBlockDelayMs = 1000 * this.clock.secondsPerSlot * (this.opts?.afterBlockDelaySlotFraction ?? 0);
```

Comment:
Since this is delicate value with a lot of background, please add a constant at the top of the file with context
```
/**
 * Explain the history of the value, why it was 1/6, why it's now 0. Link to relevant issues for the constant
 */
const DEFAULT_AFTER_BLOCK_DELAY_SLOT_FRACTION = 0
```